README

```diff
@@ -0,0 +1,19 @@
+hc-ansible-deployment is end-to-end HC deployment playbook which is using both gluster-ansible role and ovirt-ansible-hosted-engine-setup role.
```

Comment:
Gobinda we need to re-organize this a bit. Putting the playbooks in a directory called playbooks/hc/

Reply:
Done.